src/main/java/edu/harvard/iq/dataverse/DatasetVersion.java

```diff
@@ -1342,7 +1342,7 @@ public List<String[]> getGeographicCoverage() {
                     }
                     geoCoverages.add(coverageItem);
                 }
-
+                break;
```

Comment:
I'm not sure why this `break` was removed.

Reply:
Not removed - added. Once we've found the geographicCoverage field, we don't need to go through the rest of the fields.

Reply:
Whoops, added. I just wanted to make sure it was intentional. Thanks.